.github/data/agent-framework/resources/agent.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: kagent.dev/v1alpha1
```

Comment:
a work item for later: we have these agents packaged up as Helm charts, so we might want to use that, so we don't have a drift between the agents here and in the yaml.